builtin/logical/pki/path_acme_test.go

```diff
@@ -396,6 +397,29 @@ func TestAcmeAccountsCrossingDirectoryPath(t *testing.T) {
 	// swallows the error we are sending back to a no account error
 }
 
+// TestAcmeDisabledWithEnvVar verifies if VAULT_DISABLE_PUBLIC_ACME is set that we completely
+// disable the ACME service
+func TestAcmeDisabledWithEnvVar(t *testing.T) {
+	os.Setenv("VAULT_DISABLE_PUBLIC_ACME", "true")
+	defer os.Unsetenv("VAULT_DISABLE_PUBLIC_ACME")
+
+	t.Parallel()
```

Comment:
Just checking in case you hadn't considered, but typically tests which change the environment variable state are unsafe to run in parallel. Probably want to double-check that this is okay.

Reply:
Yup you are correct, this might badly influence other tests, removed the t.Parallel() call. Thanks!